rspec-mode.el

```diff
@@ -225,6 +225,13 @@ info, are considered errors."
   :type 'boolean
   :group 'rspec-mode)
 
+(defcustom rspec-holder-dirs '("app" "lib")
```

Comment:
- `rspec-container-dirs`?
- `rspec-injected-dirs`, because their contents are injected directly into `spec`?
- `rspec-psuedo-root-dirs`?
- `rspec-applike-dirs`